### ISSUE TITLE
No feedback when OpenIOC import does not validate

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#2197]
When populating a MISP event using OpenIOC this will silently fail when the OpenIOC file does not validate. CakePHP stores validation errors in `$this->Event->validationErrors` however the MISP function `_addIOCFile()` never checks the return value of `saveAssociated()` or the `validationErrors`.

One side effect of using `saveAssociated()` is that if one attribute fails to validate, none of them will be saved. In MISP the OpenIOC import seems to have worked, but none of the attributes are actually added to the event.

**Comment by truckydev** [comment]
Have you try to import your ioc with mudule ?

**Comment by truckydev** [comment]
?

**Comment by RichieB2B** [comment]
No, I will give that a try next time I need to import an IOC file.
